Dockerfile.rhel7

```diff
@@ -4,6 +4,7 @@ COPY . .
 RUN hack/build-go.sh
 
 FROM registry.svc.ci.openshift.org/ocp/4.0:base
+RUN yum install -y rsync
```

Comment:
@smarterclayton is this an allowed dockerfile change (and will it work w/ our build system?)